src/server.cc

```diff
@@ -558,8 +558,15 @@ void Server::cron() {
     if (storage_->IsClosing()) continue;
```

Comment:
@git-hulk here we already check db is closing, are you sure kvrock would execute the next code

Reply:
Yes, there still have the window between `closing` and the db pointer is ready, since we set it to `true` at the begin of the open db.

Reply:
ok